src/evaluate.py

```diff
@@ -312,6 +318,25 @@ def _write_winograd_preds(
             preds_fh.write("{0}\n".format(json.dumps(out_d)))
 
 
+def _write_boolq_preds(
```

Comment:
Verify that this is compatible with the eval server.

Reply:
Confirmed!